Framework/API/inc/MantidAPI/ImplicitFunctionParameterParser.h

```diff
@@ -51,10 +51,9 @@ namespace API {
 /*
  * Deletion policy for unique pointers.
  */
-template <typename T> class DLLExport DeleterPolicy {
-public:
-  void operator()(T *pParser) { delete pParser; }
-};
+template <typename T> class DLLExport DeleterPolicy{public : void operator()(T *pParser){delete pParser;
```

Comment:
This looks like it's made the formating worse. It seems to have done the opposite to what was done in `IFunction.h`